test/rulesets/Base/base.jl

```diff
@@ -174,4 +174,21 @@
         test_frule(Base.literal_pow, ^, 3.5, Val(3))
         test_rrule(Base.literal_pow, ^, 3.5, Val(3))
     end
+
+    @testset "Float conversions" begin
+        for f in (float, Float32, Float64, AbstractFloat)
+            test_frule(f, 1.2; rtol=1.0e-3, atol=1.0e-3)
+            test_frule(f, 1⊢1.7; rtol=1.0e-3, atol=1.0e-3)
+            test_rrule(f, 1.2; rtol=1.0e-3, atol=1.0e-3)
+
+            # test_rrule doesn't like integers, so test that case manually
+            let y = rand(Int), x = randn()
+                @test rrule(f, y)[2](x)[2] == x
```

Comment:
Really? seems bad that it can't handle it even with a `\vdash`
Can you open a issue on CRTU?

Reply:
might just need updating to CRTU 1.0?

Reply:
```
test_rrule: Float64 on Int64: Error During Test at /home/keno/.julia/packages/ChainRulesTestUtils/8380y/src/testers.jl:191
  Got exception outside of a @test
  InexactError: Int64(0.99)
  Stacktrace:
    [1] Int64
      @ ./float.jl:812 [inlined]
    [2] convert
      @ ./number.jl:7 [inlined]
    [3] setindex!
      @ ./array.jl:937 [inlined]
    [4] (::FiniteDifferences.var"#62#64"{Int64, ComposedFunction{ComposedFunction{ComposedFunction{typeof(first), typeof(FiniteDifferences.to_vec)}, ChainRulesTestUtils.var"#fnew#42"{ChainRulesTestUtils.var"#call#50"{NamedTuple{(), Tuple{}}}, Tuple{DataType, Int64}, Tuple{Bool, Bool}}}, FiniteDifferences.var"#Real_from_vec#20"}, Vector{Int64}})(ε::Float64)
      @ FiniteDifferences ~/.julia/packages/FiniteDifferences/W3rQO/src/grad.jl:18
```
on CRTU 1.0